heliopy/data/mms.py

```diff
@@ -117,7 +124,9 @@ def fgm_survey(probe, starttime, endtime):
     daylist = util._daysplitinterval(starttime, endtime)
     dirs = []
     fnames = []
-    extension = '.cdf'
+    #don't need so much string munging since we're asking SDC for things
+    #extension = '.cdf'
```

Comment:
Can you leave this line in, and strip `.cdf` off the end of the filename later (line 145)? I think the downloading code needs to know the extension so it can automatically extract units from the cdf file.

Reply:
Sure, we can do that - another approach would be letting the downloading code read the extension off a preformed name, but for now we can just take filename = filename[:-4]